br.com.wiselabs.simplexity.yml

```diff
@@ -0,0 +1,61 @@
+app-id: br.com.wiselabs.simplexity
+runtime: org.freedesktop.Platform
+runtime-version: '24.08'
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: '24.08'
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.node20
+command: run.sh
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=pulseaudio
+  - --share=network
+  - --device=dri
+  - --filesystem=home
```

Comment:
why does this need home permission?

Reply:
To support drag and drop file to Perplexity attachment area.